### ISSUE TITLE
using errors/validation is more complicated that it seems it should be (from a beginner or someone unfamiliar with the inner workings of the module perspective)

### ISSUE THREAD

**Issue by 1xch** [laurihy/angular-payments#37]
I'm admittedly a novice..but I can't seem to get a grasp on accessing field validation. 

I'm using the example form, so for example:

```
<div class="span3">
                <label for="">Card number</label>
                <input type="text"
                       class="input-block-level"
                       ng-model="number"
                       payments-validate="card"
                       payments-format="card"
                       payments-type-model="type"
                       ng-class="formName.number.$card.type"/>
```

and adding an error 

```
                <div class="error" ng-show="formName.number.$dirty">XXXXXXXXXXXX</div>
            </div>
```

seems as though I've tried:

number.$dirty
formName.number.$dirty
formName.number.$card.type.number.$dirty
formName.$card.type.$dirty

etc. and so on....I can't seem to trigger ng-show. From a novice js perspective the wording in the readme is opaque, this seems more painful than should be. I'm sure its easier to understand with more experience in js, but right now for me not so much and the example & documentation are not helping and rather than continue to flail around, I thought I would ask, and then I'll send a pull with an updated example because this is like 2/3 of this repo that is diffucult to understand on a drop-in-beginner basis, and would help making your project amenable to more people (I suppose). 

tl;dr:  Having an example that incorporates usage of 2/3 of the library would be very helpful to people trying to use this library. I'd be willing to push that out if I can figure out whats going on here that I'm having issues with.

answer seems straightforward with naming the field....supposing guessing at other parts will take some time on my effort:

```
div class="span3">
                <label for="">Card number</label>
                <input type="text"
                       name="cccardnumber"
                       class="input-block-level"
                       ng-model="number"
                       payments-validate="card"
                       payments-format="card"
                       payments-type-model="type"
                       ng-class="formName.number.$card.type"/>
```

and adding an error 

```
                <div class="error" ng-show="formName.cccardnumber.$dirty && formName.cccardnumber.$invalid">XXXXXXXXXXXX</div>
            </div>
```

still this is something that would make the example clearer to users
